website/www/site/content/en/blog/beam-2.46.0.md

```diff
@@ -0,0 +1,229 @@
+---
+title:  "Apache Beam 2.46.0"
+date:   2023-03-08 13:00:00 -0500
+categories:
+  - blog
+  - release
+authors:
+  - damccorm
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.46.0 release of Beam.
+This release includes both improvements and new functionality.
+See the [download page](/get-started/downloads/#2460-2023-03-08) for this release.
+
+<!--more-->
+
+For more information on changes in 2.46.0, check out the [detailed release notes](https://github.com/apache/beam/milestone/9?closed=1).
+
+## Highlights
+
+* Java SDK containers migrated to [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin)
+  as a base. This change migrates away from the deprecated [OpenJDK](https://hub.docker.com/_/openjdk)
+  container. Eclipse Temurin is currently based upon Ubuntu 22.04 while the OpenJDK
+  container was based upon Debian 11.
+* RunInference PTransform will accept model paths as SideInputs in Python SDK. ([#24042](https://github.com/apache/beam/issues/24042))
+* RunInference supports ONNX runtime in Python SDK ([#22972](https://github.com/apache/beam/issues/22972))
+* Tensorflow Model Handler for RunInference in Python SDK ([#25366](https://github.com/apache/beam/issues/25366))
+* Java SDK modules migrated to use `:sdks:java:extensions:avro` ([#24748](https://github.com/apache/beam/issues/24748))
+
+## I/Os
+
+* Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
```

Comment:
Minor: remove these placeholders, other than this LGTM

Reply:
Done, good catch